roles/common_charts/tasks/cert-manager-cluster-issuers.yml

```diff
@@ -0,0 +1,24 @@
+- name: Create cluster issuer temp file
+  tempfile:
+    state: file
+    suffix: cluster_issuer
+  register: cluster_issuer_tempfile
+  changed_when: false
+
+- name: Template cluster issuer file
+  template:
+    src: "{{ item }}"
+    dest: "{{ cluster_issuer_tempfile.path }}"
+  diff: no
+  changed_when: false
+
+- name: Debug file
```

Comment:
this can probably be cleaned up as its just debug code?

Reply:
ah yeah, missed that one, good catch